### PR TITLE
PPC/UCS: removed extra check for __ppc_get_timebase_freq

### PR DESCRIPTION
**Pull request by hoopoepg**
- removed double check for __ppc_get_timebase_freq function

### CODE REVIEW

File: config/m4/sysdep.m4 [hunk 1]
```diff
@@ -104,9 +104,7 @@ AC_CHECK_HEADERS([sys/platform/ppc.h])
 #
 # PowerPC query for getting TB frequency
 #
-AC_CHECK_DECL([__ppc_get_timebase_freq],
-              [AC_CHECK_FUNCS([__ppc_get_timebase_freq])], [],
-              [#include <sys/platform/ppc.h>])
```
Comment (yosefe) [inline config/m4/sysdep.m4:109]: why removed the include?

File: config/m4/sysdep.m4 [hunk 2]
```diff
@@ -104,9 +104,7 @@ AC_CHECK_HEADERS([sys/platform/ppc.h])
...
+AC_CHECK_FUNCS([__ppc_get_timebase_freq])
```
Comment (dmitrygx) [inline config/m4/sysdep.m4:107]: if I'm not mistaken it is worth checking for declaration and then for function itself

**Comment by hoopoepg** [comment]
AC_CHECK_DECL or AC_CHECK_FUNCS is enough to check existing API in libc, not both. I used AC_CHECK_FUNCS just because in the source file used HAVE_func macro to evaluate macro result

File: config/m4/sysdep.m4 (hunk 2 above)
Comment (dmitrygx) [reply config/m4/sysdep.m4:107]: I've got the following problems:
https://github.com/openucx/ucx/pull/7619#discussion_r743065020
https://github.com/openucx/ucx/pull/7619#discussion_r743075284

Comment (hoopoepg) [reply config/m4/sysdep.m4:107]: AC_CHECK_FUNCS checks for function in standard libraries, we definitely have to include system headers to source file to use function.
as I can see ```sys/platform/ppc.h``` headed is part of libc package and must correspond to library

Comment (dmitrygx) [reply config/m4/sysdep.m4:107]: sorry, this is a change for `__ppc_get_timebase_freq`, not for `__ppc_get_timebase`

**Review by dmitrygx — APPROVED** [review]

File: config/m4/sysdep.m4 (hunk 2 above)
Comment (hoopoepg) [reply config/m4/sysdep.m4:107]: oh, I see - __ppc_get_timebase is declared as inline, but __ppc_get_timebase_freq - is libc call

Comment (hoopoepg) [reply config/m4/sysdep.m4]: I bit more cleaned configs.
now it checks more natively :)

**Review by dmitrygx — APPROVED** [review]

**Comment by hoopoepg** [comment]
@yosefe ok to merge?

**Review by brminich — APPROVED** [review]

**Pull request merged**